cou/commands.py

```diff
@@ -93,8 +93,9 @@ def get_subcommand_common_opts_parser() -> argparse.ArgumentParser:
         action="count",
         dest="verbosity",
         help="Increase logging verbosity in STDOUT. Multiple 'v's yield progressively "
-        "more detail (up to 4).\nNote that this doesn't affect the verbosity in logfile, "
-        "which will always have the maximum verbosity.",
+        "more detail (up to 4).\nNote that by default cou use debug level at the logfile."
+        "However, at the maximum verbosity it will include debug logs from libjuju and "
+        "websockets that can generate a big logfile.",
```

Comment:
The description here suggests that by default only COU logs at debug (and up) level will be included in the logfile. But that doesn't align with the actual implementation where external logs that are not in debug level will also be included in general cases. We should change the wording here to reflect this.

Reply:
Do you have a suggestion on how to re word it?